src/test/java/be/seeseemelk/mockbukkit/entity/PlayerMockTest.java

```diff
@@ -1371,4 +1372,29 @@ public void testPlayerSpawnParticle_Incorrect_DataType()
 		});
 	}
 
+	@Test
+	public void getAddress_Constructor()
+	{
+		PlayerMock player = server.addPlayer();
+		assertNotNull(player.getAddress());
+	}
+
+	@Test
+	public void setAddress()
+	{
+		PlayerMock player = server.addPlayer();
+		InetSocketAddress address = new InetSocketAddress("192.0.2.78", 25565);
+		player.setAddress(address);
+		assertEquals(address, player.getAddress());
+	}
+
+	@Test
+	public void getAddress_NullWhenNotOnline()
```

Comment:
Just a note: As of Junit 5 test methods are supposed to have no access modifier by convention.
So no need to add `public` to these. I know, we still have them in many places (including classes) as we used to be running Junit 4 in the past.

Just keep that in mind.

Reply:
My bad. Once we establish naming conventions for tests we've gotta go through and rename all of them and fix the modifiers so everything's consistent.